doc/md/atlas-schema/hcl-types.md

```diff
@@ -1170,6 +1170,104 @@ table "t" {
 }
 ```
 
+### User-defined types
+
+#### Alias Types
```

Comment:
Wrap it with driver tabs maybe? Because I will add pg to it as well

Reply:
I think this document is per driver already.